bittide/src/Bittide/ClockControl/Si539xSpi.hs

```diff
@@ -278,3 +303,14 @@ si539xSpiDriver SNat incomingOpS miso = (fromSlave, decoderBusy, spiOut)
       | otherwise = Nothing
 
 {-# NOINLINE si539xSpiDriver #-}
+
+holdTrue ::
```

Comment:
Could you add documentation? Also; this doesn't really scale well given that it is based on a vector. This is okay, but should be noted.